crates/nu-command/src/database/values/sqlite.rs

```diff
@@ -428,10 +436,89 @@ pub fn open_sqlite_db(path: &Path, call_span: Span) -> Result<Connection, ShellE
 fn run_sql_query(
     conn: Connection,
     sql: &Spanned<String>,
+    params: NuSqlParams,
     ctrlc: Option<Arc<AtomicBool>>,
 ) -> Result<Value, SqliteError> {
     let stmt = conn.prepare(&sql.item)?;
-    prepared_statement_to_nu_list(stmt, sql.span, ctrlc)
+
+    prepared_statement_to_nu_list(stmt, params, sql.span, ctrlc)
+}
+
+pub fn nu_unit_value_to_sql_value(value: &Value) -> Result<&dyn ToSql, ShellError> {
+    const NONE: Option<bool> = None;
+
+    match value {
+        Value::Bool { val, .. } => Ok(val),
+        Value::Int { val, .. } => Ok(val),
+        Value::Float { val, .. } => Ok(val),
+        Value::String { val, .. } => Ok(val),
+        Value::Binary { val, .. } => Ok(val),
+        Value::Nothing { .. } => Ok(&NONE),
+
+        // To consider:
+        Value::Date { val, .. } => Ok(val),
+        // Value::Filesize { val, .. } => todo!(),
```

Comment:
a function like this already exists in into_sqlite that supports filesize and duration. it also handles string differently. maybe you should move that function here and use it?

Reply:
There could be several functions in `into_sqlite` that may fit better in values/sqlite.rs where others can use them if needed. The key here is to not have duplicate functions and not reinvent the wheel if it already exists. Thanks!

Reply:
Of course, for consistency. I imagined the function already existed but I didn't want to sleuth, and the ToSql trait did all the legwork anyways. Didn't imagine that filesize and durations made sense as sqlite values. I'll check it out.

Reply:
Now it mostly operates on owned values.